docs/standard/security/cross-platform-cryptography.md

```diff
@@ -174,47 +207,51 @@ The <xref:System.Security.Cryptography.ECDiffieHellman> class doesn't return the
 * HMAC(Z, prepend || Z || append)
 * Tls11Prf(label, seed)
 
+"Raw" key derivation was introduced in .NET 8.
+
 ECDH key curves are defined by the OS libraries and are subject to their limitations.
 
-| Elliptic Curve                     | Windows 10    | Windows 7 - 8.1 | Linux         | macOS         |
-|------------------------------------|---------------|-----------------|---------------|---------------|
-| NIST P-256 (secp256r1)             | ✔️           | ✔️              | ✔️           | ✔️            |
-| NIST P-384 (secp384r1)             | ✔️           | ✔️              | ✔️           | ✔️            |
-| NIST P-521 (secp521r1)             | ✔️           | ✔️              | ✔️           | ✔️            |
-| brainpool curves (as named curves) | ✔️           | ❌              | ⚠️<sup>1</sup>| ❌           |
-| other named curves                 | ⚠️<sup>2</sup>| ❌             | ⚠️<sup>1</sup>| ❌           |
-| explicit curves                    | ✔️           | ❌              | ✔️           | ❌            |
-| Export or import as explicit       | ✔️           | ❌<sup>3</sup>  | ✔️           | ❌<sup>3</sup>|
+| Elliptic Curve                     | Windows 10     | Windows 7 - 8.1 | Linux          | macOS           | iOS            | Android        |
+|------------------------------------|----------------|-----------------|----------------|-----------------|----------------|----------------|
+| NIST P-256 (secp256r1)             | ✔️             | ✔️              | ✔️             | ✔️             | ✔️             | ✔️             |
+| NIST P-384 (secp384r1)             | ✔️             | ✔️              | ✔️             | ✔️             | ✔️             | ✔️             |
+| NIST P-521 (secp521r1)             | ✔️             | ✔️              | ✔️             | ✔️             | ✔️             | ✔️             |
+| Brainpool curves (as named curves) | ✔️             | ❌              | ⚠️<sup>1</sup> | ❌             | ❌             | ⚠️<sup>4</sup> |
+| Other named curves                 | ⚠️<sup>2</sup> | ❌              | ⚠️<sup>1</sup> | ❌             | ❌             | ⚠️<sup>4</sup> |
+| Explicit curves                    | ✔️             | ❌              | ✔️             | ❌             | ❌             | ✔️             |
+| Export or import as explicit       | ✔️             | ❌<sup>3</sup>  | ✔️             | ❌<sup>3</sup> | ❌<sup>3</sup> | ✔️             |
```

Comment:
Would it be good (or bad) to call out curve25519 as special here (in the ECDH table)?  IIRC, it works on Windows because Windows uses oid.FriendlyName, and nowhere else because they use oid.Value (and curve25519 has no registered OID)

Reply:
I would lean toward "no" because I don't think it does what people want it to do: they want X25519 for the DH function. i.e. X25519 always uses point compression (hence the X) and using that with `ECDiffieHellman` in a meaningful way would be tricky.